src/components/job-row.tsx

```diff
@@ -1,19 +1,16 @@
 import React, { useState } from 'react';
 
 import { JupyterFrontEnd } from '@jupyterlab/application';
-
-import DownloadIcon from '@mui/icons-material/Download';
-import StopIcon from '@mui/icons-material/Stop';
-import IconButton from '@mui/material/IconButton';
-import { Stack, TableCell, TableRow } from '@mui/material';
-
-import { ConfirmDeleteIcon } from './confirm-delete-icon';
+import { ConfirmDeleteIcon, ConfirmIcon } from './confirm-icons';
 import { JobFileLink } from './job-file-link';
-
 import { CommandIDs } from '..';
 import { Scheduler } from '../handler';
 import { useTranslator } from '../hooks';
 import { ICreateJobModel } from '../model';
+import DownloadIcon from '@mui/icons-material/Download';
+import StopIcon from '@mui/icons-material/Stop';
+import IconButton from '@mui/material/IconButton';
+import { Stack, TableCell, TableRow } from '@mui/material';
```

Comment:
```suggestion
import { IconButton, Stack, TableCell, TableRow } from '@mui/material';
```

You can import all the components you need from `@mui/material`.

Reply:
Thanks. Reminder for me to doublecheck imports, not just 100% rely on auto imports